### PR TITLE
Parent-dash-contact-modal

### PR DESCRIPTION
**Pull request by Robert-Lark**
# Description

This pull request relates to the contact modal that pops up when the contact us button is clicked on the support page. It is one of three options to display the 'contact us' form that we will present to the stakeholders.

I used a couple of packages to handle the functionality. 

Emailjs handles the sending of data to a dedicated email address (storysquaddemo@gmail.com), it also handles form validation

Toastify handles the success notification. 

I decided not to use ant-design here as it wasn't compatible with either package and I felt the functionality that the packages brought was superior to what ant-design brought. 

UseSelector is used to grab the parent's username and email address from the reducer to pre-populate the form. This allows the user to not have to worry about writing their name and email into the form (they can, however, edit it if necessary).

Fixes #

- What work was done?
 Implementation of a form and submit functionality. 
- Why was this work done?
Give users a way to submit their own queries to customer support. 
- What feature / user story is it for?
As a user I would like a clear UI to access common questions and contact staff.
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "react": "^16.13.1",
     "react-datepicker": "^3.4.1",
     "react-dom": "^16.13.1",
```
Comment (Robert-Lark) [inline package.json:27]: removed react-hook-form as emailjs has inbuilt form validation.

File: src/components/pages/SupportPage/SupportPageContainer.js [hunk 2]
```diff
@@ -1,38 +1,27 @@
...
+      pauseOnHover: true,
+      draggable: false,
+      className: 'submit-feedback success',
+      toastId: 'notifyToast',
+    });
+  };
 
   return (
     <>
-      <Header title="Support" displayMenu={true} />
```
Comment (Robert-Lark) [inline src/components/pages/SupportPage/SupportPageContainer.js:10]: moved this code into its own file - renderSupportPageModal

File: src/styles/less/SupportPage.less [hunk 3]
```diff
@@ -1,14 +1,14 @@
 @import '../less/variables.less';
 
-.container {
```
Comment (Robert-Lark) [inline src/styles/less/SupportPage.less:3]: these classnames were too generic so I renamed them.

**Review by StefxM — APPROVED** [review]
its really coming along , looks good !

**Review by Patrick-G-Oliver — APPROVED** [review]
Great to see the toastify and emailjs npm packages incorporated and the less classes renamed for specificity.

**Pull request merged**